test/result/function/func_datetime_weekday.result

```diff
@@ -10,36 +10,23 @@ insert into t1 values(71, 1, 2, 34, 5, 5.5, 341.13, 15.314, "2012-03-22", "2013-
 insert into t1 values(1, 1, 21, 4, 54, 53.5, 431.13, 14.394, "2011-03-12", "2015-03-12 10:03:12", "2002-03-12 13:03:12", "afbc", "dct5f");
 insert into t1 values(1, 71, 2, 34, 5, 5.5, 31.313, 124.314, "2012-01-12", "2019-03-12 10:03:12", "2013-03-12 13:03:12", "3abd1c", "dcvf");
 select weekday(a),weekday(b),weekday(c),weekday(d),weekday(e),weekday(f) from t1;
-weekday(a)	weekday(b)	weekday(c)	weekday(d)	weekday(e)	weekday(f)
-null	null	null	null	null	null
-null	null	null	null	null	null
-null	null	null	null	null	null
-null	null	null	null	null	null
+Function 'weekday' with parameters [TINYINT] will be implemented in future version.
```

Comment:
You've introduced some new errors, please tag with a new issue instead of modifying expected results.